src/ODE.jl

```diff
@@ -300,11 +311,11 @@ function ode23s(F, y0, tspan; reltol = 1.0e-5, abstol = 1.0e-8,
         F2 = F(t + h, ynew)
         k3 = W\(F2 - e32*(k2 - F1) - 2*(k1 - F0) + T )
 
-        err = (abs(h)/6)*norm(k1 - 2*k2 + k3) # error estimate
-        delta = max(reltol*max(norm(y),norm(ynew)), abstol) # allowable error
+        threshold = abstol/reltol # error threshold
+        err = (abs(h)/6)*norm((k1 - 2*k2 + k3)./max(max(abs(y),abs(ynew)),threshold),Inf) # error estimate
```

Comment:
err has suddenly changed meaning here - it is now what it was before divided by something else. Should it have another name, too?